roles/test_operator/tasks/runners/tobiko_runner.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Since Tobiko may include disruptive tests (faults), it is better to execute it at the end
+- name: Run tobiko job
+  vars:
+    _test_operator_tobiko_namespace: "{{ _stage_test_vars.cifmw_test_operator_tobiko_namespace | default(cifmw_test_operator_tobiko_namespace) }}"
```

Comment:
Just a though. Something like this won't work? We would not need to update this portion of code if there is a new var.
```
- name: Overwrite global_vars with stage_vars
  with_dict: "{{ vars }}"
  when: item.key.startswith('cifmw_test_operator')
  set_fact:
    "{{ item.key | replace('cifmw', '') }}": "{{ _stage_test_vars[item.key] | default(item.value) }}"
```

Reply:
this would only override vars assuming they are set in the {{ vars }}, assuming you want to set a stage var that doesn't exist as a regular var starting with 'cifmw_test_operator' it won't get overridden for the next stage and would just stay as it is. For example if I have:
```
cifmw_test_operator_stages:
  - name: tempest-basic-tests
    type: tempest
    test_vars:
        cifmw_test_operator_some_basic_test_var: 'some important value'
  - name: tempest-more-tests
    type: tempest
 ```
`cifmw_test_operator_some_basic_test_var`  would be used in the `tempest-more-tests` stage as well if it has no default value or if it is not set in the var files we are using with -e.